leaderboard/utils/formatter.py

```diff
@@ -229,9 +232,16 @@ def format_pr_contributions(
             merged_by_id = pr_node["mergedBy"]["id"]
 
         labels = []
+        do_not_continue = False
         for edge in pr_node["labels"]["edges"]:
+            # if invalid label, do not continue
+            if edge["node"]["name"] == "invalid":
```

Comment:
If we go by the official Hacktoberfest rules, it's both `invalid` and `spam` labels.